spec/index.bs

```diff
@@ -4233,10 +4233,10 @@ dictionary GPUTextureDescriptor
         Formats in this list must be [=texture view format compatible=] with the texture format.
 
         <div algorithm data-timeline=const>
-            Two {{GPUTextureFormat}}s |format| and |viewFormat| are <dfn dfn for="">texture view format compatible</dfn> if:
+            Two {{GPUTextureFormat}}s |format| and |viewFormat| are <dfn dfn for="">texture view format compatible</dfn> for a given |device| if:
```

Comment:
This text is good, but the "call site" in `validating GPUTextureDescriptor` should explicitly say `... must be [=texture view format compatible=] for device |this|.`

actually nit: probably s/for/on/

Reply:
I'll apply this change, will only take a second.